core/lib/core/next_actions.ex

```diff
@@ -19,6 +19,14 @@ defmodule Core.NextActions do
     |> Enum.map(&to_view_model(url_resolver, &1))
   end
 
+  @doc """
+  """
+  def next_best_action!(url_resolver, %User{} = user, content_node \\ nil)
+      when is_function(url_resolver) do
+    list_next_actions(url_resolver, user, content_node)
+    |> List.first()
```

Comment:
This could be altered to  use Ecto [one](https://hexdocs.pm/ecto/Ecto.Repo.html#c:one/2). That would be slightly more efficient than getting all the items.

Reply:
👍